src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,12 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.TextUi;
+
+public class ListCommand extends Command {
```

Comment:
Perhaps you could add some Javadocs for all of your classes? I noticed that many of the Command classes do not have javadocs